scripts/install.sh

```diff
@@ -19,8 +19,9 @@ set -ev
 pip install tox
 
 # App Engine tests require the App Engine SDK.
-if [[ "${TOX_ENV}" == "gae" && ! -d ${GAE_PYTHONPATH} ]]; then
-    python scripts/fetch_gae_sdk.py `dirname ${GAE_PYTHONPATH}`
+if [[ "${TOX_ENV}" == "gae" || "${TOX_ENV}" == "cover" ]]; then
+    pip install git+https://github.com/GoogleCloudPlatform/python-repo-tools.git
```

Comment:
1. We intentionally don't publish it to PyPI because it's never supposed to be used outside of well, this exact situation.
2. Doesn't matter. In python-docs-samples, it's installed in the virtualenv that nox creates for each test session.